### DISCUSSION TITLE
[8 febbraio 2024] Calcolabilità

### DISCUSSION THREAD

**Discussion by Antaarees** [sapienzastudentsnetwork/automi-calcolabilita-complessita#24] [Calcolabilità]
![20240213_113332](https://github.com/sapienzastudentsnetwork/acc2324/assets/19594279/74603bf8-ad21-4441-8298-6f14e489a36e)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo le seguenti due soluzioni presenti nei miei appunti:

![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/65043207/7e8267a6-7094-46e3-b079-403a8a7943cb)

----

Le due dimostrazioni richiesta nel secondo punto sono presenti sul libro di testo

**Comment by CasuFrost** [comment]
Per il secondo punto (una dimostrazione concisa)
![image](https://github.com/user-attachments/assets/932e27ca-eee9-4dc0-ad3b-35ccd4cbb7c2)
